lib/hashdiff/util.rb

```diff
@@ -48,12 +48,12 @@ def self.count_nodes(obj)
 
   # @private
   #
-  # decode property path into an array
+  # decode a property path into an array
   # @param [String] path Property-string
```

Comment:
I'm fine to remove this as we are moving to array representation, this is no longer relevant.